skyportal/utils/offset.py

```diff
@@ -2,6 +2,7 @@
 import os
 import datetime
 import warnings
+import json
```

Comment:
```suggestion
```
json import isn't used - causing the failure on GA

Reply:
Done